src/Executables/Auth/PacketHandlers/LoginRequestHandler.cs

```diff
@@ -4,9 +4,32 @@
 using QuantumCore.Auth.Packets;
 using QuantumCore.Caching;
 using QuantumCore.Core.Utils;
+using System.ComponentModel;
+using QuantumCore.API.Core.Models;
 
 namespace QuantumCore.Auth.PacketHandlers;
 
+public enum LoginFailedBecause
```

Comment:
I love enums. Thanks.
If the description is used only for developers please use the `/// <summary>`